src/app/passport/controllers/validate.js

```diff
@@ -33,33 +37,47 @@ class ValidateController extends BaseController {
         passport_session_id: req.session.passportSessionId,
       };
 
-      logger.info("calling authorize lambda", { req, res });
-      const apiResponse = await axios.post(
-        `${API_BASE_URL}${API_AUTHORIZE_PATH}${queryParams}`,
+      logger.info("calling check-passport lambda", { req, res });
+      const checkPassportResponse = await axios.post(
+        `${API_BASE_URL}${API_CHECK_PASSPORT_PATH}${queryParams}`,
         attributes,
         { headers: headers }
       );
 
-      const code = apiResponse?.data?.code?.value;
+      if (checkPassportResponse.data?.result === "retry") {
+        req.sessionModel.set("showRetryMessage", true);
```

Comment:
Maybe add a log here to track the retries?

Reply:
Added log